cmd/cluster/support/post.go

```diff
@@ -1,319 +1,203 @@
 package support
 
 import (
-	"encoding/json"
+	"bytes"
 	"fmt"
-	"log"
-	"net/http"
-	"net/url"
-	"os"
-	"strings"
-
-	"github.com/openshift-online/ocm-cli/pkg/arguments"
 	"github.com/openshift-online/ocm-cli/pkg/dump"
 	sdk "github.com/openshift-online/ocm-sdk-go"
-	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
-	"github.com/openshift/osdctl/internal/support"
-	"github.com/openshift/osdctl/internal/utils"
-	"github.com/openshift/osdctl/internal/utils/globalflags"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	slv1 "github.com/openshift-online/ocm-sdk-go/servicelogs/v1"
 	ctlutil "github.com/openshift/osdctl/pkg/utils"
 	"github.com/spf13/cobra"
-	"k8s.io/cli-runtime/pkg/genericclioptions"
-	cmdutil "k8s.io/kubectl/pkg/cmd/util"
-)
-
-var (
-	template                                                string
-	templateParams, userParameterNames, userParameterValues []string
+	"os"
 )
 
 const (
-	defaultTemplate = ""
+	LimitedSupportSummary         = "Cluster is in Limited Support due to unsupported cluster configuration"
```

Comment:
I think the old workflow had unsupported **cluster** configuration and unsupported **cloud** configuration.
So probably this needs to be configurable, because not everything is the cluster being misconfigured.

Reply:
Good catch - I'll add a flag to account for this.